### ISSUE TITLE
request to transfer ownership of Pursuit package

### ISSUE THREAD

**Issue by peterbecich** [coot/purescript-argonaut-aeson-generic#18]
@coot , would you consider transferring ownership of the Pursuit package https://pursuit.purescript.org/packages/purescript-argonaut-aeson-generic to me?

We have been discussing this here: https://github.com/coot/purescript-argonaut-aeson-generic/pull/17#issuecomment-1304328361

My fork is: https://github.com/peterbecich/purescript-argonaut-aeson-generic

The transfer instructions: https://github.com/purescript/registry-dev/blob/master/SPEC.md#54-transfer-a-package-authenticated

The payload might be:
```
{
  "name": "argonaut-aeson-generic",
  "newLocation": {
    "githubOwner": "peterbecich",
    "githubRepo": "purescript-argonaut-aeson-generic"
  }
}
```

Thank you

**Comment by StephenWakely** [comment]
It looks like the current owner is @kl0tl.

**Comment by peterbecich** [comment]
Email sent.

@kl0tl , please consider uploading a new Pursuit release from tag `v0.4.0` here https://github.com/peterbecich/purescript-argonaut-aeson-generic/releases/tag/v0.4.0  It's not necessary to transfer ownership to me, but that would alternatively work. Thanks

**Comment by JordanMartinez** [comment]
I think @kl0tl is no longer with us. Maybe due to Covid? Here's why I think this:

kl0tl was the one who initially led the effort for supporting ES Modules. kl0tl also submitted a number of PRs that migrated core libraries among others over to ES modules, many of which we merged when we made those breaking changes. We also had multiple Discourse posts stating that were were making this change and had the weekly call on Discord to coordinate our efforts. In other words, there were multiple ways that kl0tl could have been notified of our efforts. However, kl0tl didn't contribute any code or even say anything anywhere on Discourse, GitHub, etc, even when we pinged @kl0tl.

I found this so surprising that I emailed kl0tl using the email listed on kl0tl's commits a few months after the ES modules work was merged and the breaking changes finished. Since kl0tl started this effort, I figured kl0tl would want to know about these changes finally getting done. I never got a response back. So, I also checked the GitHub activity for that account. While there are a few blips in the past year, you'll notice that all of these "contributions" are just people merging the PRs that kl0tl submitted sometime in 2020 or so.

In other words, there has been no new activity on the account. Looking at the history of contributions made by that account, there has been a year (2019) where little occurred, but it's otherwise pretty active throughout the year for the other years. Once you hit February or Match 2021, suddenly the contributions slow to almost nothing, and I'd bet that all of the contributions beyond this point are the PR merges I described above.

I'd be glad to know that kl0tl is alive and be shown to be wrong. However, given the above evidence, I don't think kl0tl is.

**Comment by peterbecich** [comment]
@JordanMartinez , good news, I e-mailed @kl0tl yesterday at the address listed here https://github.com/kl0tl, and heard back today. All is well a.f.a.i.k.

**Comment by JordanMartinez** [comment]
Awesome!

**Comment by tonicebrian** [comment]
@peterbecich so is the transfer going to happen? Pursuit still shows the version uploaded by @kl0tl

**Comment by coot** [comment]
The pursuite package is owned by @kl0tl; I am also fine to transfer the repo itself to the maintainer, but it's not clear to me who that is.

**Comment by JordanMartinez** [comment]
Sorry for not saying this earlier. Technically, anyone can upload docs to Pursuit. So, I wouldn't quite say `kl0tl` owns the Pursuit package. Rather, that's just who last uploaded the docs. If this package made a new version, anyone could publish this docs if it hadn't been done already.

**Comment by peterbecich** [comment]
Thanks @coot and @JordanMartinez .

@coot , please make me a maintainer of the repository.

@JordanMartinez , I tried to publish a new version `0.4.0` to Pursuit as a non-owner here https://github.com/purescript/registry/issues/88#issuecomment-1321238522, targeting `peterbecich/purescript-argonaut-aeson-generic`, and it was understandably rejected. Would it succeed targeting `coot/...` as a non-owner, if the tag existed there?

Maybe I need to follow these instructions https://pursuit.purescript.org/help/authors#submitting-packages , instead of the `purescript/registry` way.

@tonicebrian , I received one e-mail response from @kl0tl and he offered to help with the package, so I requested that he transfer ownership of the Pursuit package to me. However, I have not heard back. If we determine it's necessary for @kl0tl to transfer ownership, would you send him another e-mail at the address listed here? https://github.com/kl0tl

Does anyone else have the permission to publish tag `v0.4.0` from here https://github.com/peterbecich/purescript-argonaut-aeson-generic/releases/tag/v0.4.0 ? Otherwise, do we need to request @kl0tl to transfer the Pursuit ownership to me with these instructions? https://github.com/purescript/registry-dev/blob/master/SPEC.md#54-transfer-a-package-authenticated

**Comment by tonicebrian** [comment]
Email sent.

**Comment by coot** [comment]
@peterbecich I sent you an invitation.

**Comment by peterbecich** [comment]
Thanks @coot

**Comment by peterbecich** [comment]
Next
> publish v0.4.0 to Pursuit
https://github.com/coot/purescript-argonaut-aeson-generic/issues/20

**Comment by peterbecich** [comment]
@kl0tl , please disregard our requests. It appears I have permission to publish to Pursuit now: https://github.com/purescript/registry/issues/127 Thanks

**Comment by peterbecich** [comment]
No need to transfer Pursuit package; upload was successful
@tonicebrian https://pursuit.purescript.org/packages/purescript-argonaut-aeson-generic/0.4.1
